pkg/routeagent_driver/handlers/ovn/uninstall.go

```diff
@@ -105,3 +97,16 @@ func (ovn *Handler) flushAndDeleteIPTableChains(table, tableChain, submarinerCha
 		logger.Errorf(err, "Error deleting iptable chain %q of table %q", submarinerChain, table)
 	}
 }
+
+// TODO need to be removed when the clusters are fully upgraded to new implementation.
+func (ovn *Handler) LegacyCleanup() {
```

Comment:
Where is this API called?

Reply:
This will be used by upgrade later. as a part of https://github.com/submariner-io/enhancements/issues/193